h/util/user.py

```diff
@@ -2,17 +2,19 @@
 """Some shared utility functions for manipulating user data."""
 import re
 
+from h.exceptions import InvalidUserId
+
 
 def split_user(userid):
     """Return the user and domain parts from the given user id as a dict.
 
     For example if userid is u'acct:seanh@hypothes.is' then return
     {'username': u'seanh', 'domain': u'hypothes.is'}'
 
-    :raises ValueError: if the given userid isn't a valid userid
+    :raises InvalidUserId: if the given userid isn't a valid userid
```

Comment:
I do think this is definitely an improvement!

Reply:
It might make Sentry a bit more helpful too if we see these popping out in other places.